src/Barman.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Barman {
+    private Scanner scanner = new Scanner(System.in);
+    private int ingredientsNumber;
```

Comment:
This shouldnt be a field.

Reply:
I use them in two methods. How else to do it?

Reply:
Done